controllers/controllers/workloads/cfapp_controller.go

```diff
@@ -66,7 +66,9 @@ func (r *CFAppReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 	cfApp := &korifiv1alpha1.CFApp{}
 	err := r.Client.Get(ctx, req.NamespacedName, cfApp)
 	if err != nil {
-		r.Log.Error(err, "unable to fetch CFApp")
+		if !apierrors.IsNotFound(err) {
+			r.Log.Error(err, "unable to fetch CFApp")
+		}
 		// ignore not-found errors, since they can't be fixed by an immediate requeue
 		// (we'll need to wait for a new notification), and we can get them on deleted requests
 		return ctrl.Result{}, client.IgnoreNotFound(err)
```

Comment:
The `job-task-runner` does this a little differently:
https://github.com/cloudfoundry/korifi/blob/5ecdf2a1fe595584a91c8df46712d33f08a7ea91/job-task-runner/controllers/taskworkload_controller.go#L78-L81

It logs them as `Info` and skips the `IgnoreNotFound` function call (this just does a similar check and returns `nil` when its a "not found" error).

For most of these cases I'm not sure it's even worth logging them as info, though, and I kind of like the semantics of `client.IgnoreNotFound`.